src/QMCWaveFunctions/tests/test_RotatedSPOsT.cpp

```diff
@@ -0,0 +1,911 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source
+// License. See LICENSE file in top directory for details.
+//
+// Copyright (c) 2022 QMCPACK developers.
+//
+// File developed by: Joshua Townsend, jptowns@sandia.gov, Sandia National
```

Comment:
It looks like someone is running clang-format with a column limit of 80 (vs the qmcpack default of 120). (Or is using an editor that does auto-wrapping).  While technically within the guidelines (line length is < 120), it is creating extra differences between files.